src/RegistrationEmailSentController.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/Enums',
+  'util/FormController',
+  'util/FormType'
+],
+function (Okta, Enums, FormController, FormType) {
+
+  var _ = Okta._;
+
+  return FormController.extend({
+    className: 'registration-email-sent',
+    Model: function () {
+    },
+
+    Form: {
```

Comment:
:nit Do we have to use a form here since all it has is just message that email is sent. Feels semantically incorrect since we don't have any form inputs.